eth/_utils/rlp.py

```diff
@@ -71,29 +71,20 @@ def validate_rlp_equal(obj_a: BaseBlock,
     diff = diff_rlp_object(obj_a, obj_b)
     if len(diff) == 0:
         raise TypeError(
-            "{} ({!r}) != {} ({!r}) but got an empty diff".format(
-                obj_a_name,
-                obj_a,
-                obj_b_name,
-                obj_b,
-            )
+            f"{obj_a_name} ({repr(obj_a)}) != "
```

Comment:
These two are equivalent:

```
f"{repr(obj)}"
f"{obj!r}"
```

can you update to use the `!r` syntax?

Reply:
I had no idea I could do that :) That definitely looks much better. I will update.